java/ql/src/Security/CWE/CWE-918/RequestForgery.qll

```diff
@@ -190,3 +195,117 @@ private class SpringRestTemplateUrlMethods extends Method {
     result = ma.getArgument(0)
   }
 }
+
+/** A sanitizer for request forgery vulnerabilities. */
+abstract class RequestForgerySanitizer extends DataFlow::Node { }
+
+private class HostnameSanitzingPrefix extends CompileTimeConstantExpr {
```

Comment:
Don't want to taint the PR discussion, but perhaps this is meant to be `HostnameSan(i)tizingPrefix`?

Reply:
Thanks, fixed